src/example-types/metricGroups/pivot.js

```diff
@@ -44,19 +44,19 @@ let maybeWrapWithFilterAgg = ({ query, filters, aggName }) =>
       }
 
 // Builds filters for drilldowns
-let drilldownFilters = ({ drilldowns = [], groups = [], schema, getStats }) =>
-  compactMapAsync((group, i) => {
-    let filter = lookupTypeProp(_.stubFalse, 'drilldown', group.type)
-    // Drilldown can come from root or be inlined on the group definition
-    let drilldown = drilldowns[i] || group.drilldown
-    return drilldown && filter({ drilldown, ...group }, schema, getStats)
-  }, groups)
+let drilldownFilters = ({ drilldowns = [], rows = [], schema, getStats }) =>
```

Comment:
`drilldownFilters` should be groups and not rows - it operates on both. Nothing in this method should be renamed

Reply:
Fixed